### PR TITLE
Add more detailed guidance about middle names

### PR DESCRIPTION
**Pull request by edwardhorsford**
Addressing #1354 at the request of @timpaul.

This makes clearer how to label middle names.

It does not address the issue I've seen frequently that the `Full name` field induces users to provide full legal names including middle name, when many services don't actually want this.

### CODE REVIEW

**Review by timpaul — APPROVED** [review]
Approved - thanks Ed!

File: src/patterns/names/index.md.njk [hunk 1]
```diff
@@ -49,10 +51,18 @@ If users are from outside the UK, use the labels:
...
 
+#### Middle names
+
+Only ask for middle names if your service requires them. 
+
+Use the label:
+
+- ’Middle names‘
+
+Make middle names optional as not all users have middle names. They do not need `(optional)` in the label as users should provide them if they have one - and will skip the field if they do not have one.
```
Comment (calvin-lau-gds) [inline src/patterns/names/index.md.njk:64]: Make sure middle names are optional, as not everyone has them.

The label should not include `(optional)`. Users will enter their middle names if they have them, and skip the field if they do not.

**Review by calvin-lau-gds — APPROVED** [review]

**Review by calvin-lau-gds — APPROVED** [review]

**Review by calvin-lau-gds — APPROVED** [review]

**Comment by calvin-lau-gds** [comment]
Hi @timpaul, ready for you to take another look before merging.
I've made some edits to fit the page style, but please comment away as I'm still getting the hang of writing for these pages.

**Comment by edwardhorsford** [comment]
Thanks both.

**Review by timpaul — APPROVED** [review]

**Pull request merged**